ui/components/app/name/name.stories.tsx

```diff
@@ -230,16 +230,3 @@ export const DefaultFirstPartyNameStory = {
     variation: VARIATION_MOCK,
   },
 };
-
-/**
- * Clicking the component will not display a modal to edit the name.
- */
-export const EditDisabledStory = {
-  name: 'Edit Disabled',
-  args: {
-    value: ADDRESS_MOCK,
-    type: NameType.ETHEREUM_ADDRESS,
-    variation: VARIATION_MOCK,
-    disableEdit: true,
```

Comment:
The `disableEdit` property was only being used because the component circularly referenced _and partially re-rendered_ itself. Now that the part that was re-rendered is its own component (`name-display.tsx`), we have no use for this `disableEdit` property. So this export here has been removed.